test/backend/test.py

```diff
@@ -157,6 +157,11 @@ def supports_device(cls, device):
     "test_exp_cpu",
     "test_exp_example_cpu",
 
+    # Gather Op:
```

Comment:
not finish?

Reply:
I had no way of testing locally, uncommented and will see if it goes through fine.

Reply:
It failed to recognize the test names. Will have to investigate locally first. Disabled for now. Please approve the patch, will work on it once I get my environment to successfully run any backend tests. Make fails for me on my Mac at this time